docs/src/pages/components/data-grid/overview/DataGridDemo.js

```diff
@@ -3,13 +3,27 @@ import { DataGrid } from '@material-ui/data-grid';
 
 const columns = [
   { field: 'id', headerName: 'ID', width: 90 },
-  { field: 'firstName', headerName: 'First name', width: 150 },
-  { field: 'lastName', headerName: 'Last name', width: 150 },
+  {
+    field: 'firstName',
+    headerName: 'First name',
+    width: 150,
+    editable: true,
+    disableClickEventBubbling: true,
```

Comment:
why do we need this prop?

Reply:
It stops the propagation of any click inside the cell. Without it, if you click the cell the row is selected.

<img width="600" src="https://user-images.githubusercontent.com/42154031/120682603-06d77a00-c473-11eb-923e-683553fea3f7.gif" alt="HnuGKhIGT4" style="max-width:100%;">

Reply:
we should use disableSelectionOnClick

Reply:
Updated!

Reply:
FYI I believe we should also be able to handle that with `event.stopPropagation()` directly